src/common/services/session/session.service.js

```diff
@@ -317,6 +318,25 @@ const session = /* @ngInject */ function ($cookies, $rootScope, $http, $timeout,
     }
   }
 
+  function signOutWithoutRedirectToOkta () {
+    const observable = currentRole() === Roles.public
+      ? Observable.of(true)
```

Comment:
I don't think we actually want this to be conditional. Logging out a public role will create a new customer in the EP database and ensure that the user has a brand new cart, which we want.

Reply:
Gotcha, I'll update it to always log the user out of EP.